docs/doc/03-develop/00-golang.md

```diff
@@ -10,14 +10,14 @@ description:
 * **Databend :** Make sure Databend is running and accessible, see [How to deploy Databend](/doc/deploy).
```

Comment:
Before you start, make sure you have successfully deployed Databend. For how to deploy Databend, see [How to deploy Databend](/doc/deploy).